Doc/howto/unicode.rst

```diff
@@ -103,10 +56,21 @@ a lot of tables listing characters and their corresponding code points:
    0063    'c'; LATIN SMALL LETTER C
    ...
    007B    '{'; LEFT CURLY BRACKET
+   ...
+   2167    'Ⅶ ': ROMAN NUMERAL EIGHT
```

Comment:
I don't think there should be a space next to the "Ⅶ".

Reply:
Fixed.